build/common.sh

```diff
@@ -1177,7 +1177,9 @@ function kube::release::gcs::publish() {
 
   local contents
   if [[ ${KUBE_GCS_MAKE_PUBLIC} =~ ^[yY]$ ]]; then
+    kube::log::status "Making uploaded version file public and non-cacheable."
     gsutil acl ch -R -g all:R "${publish_file_dst}" >/dev/null 2>&1 || return 1
+    gsutil setmeta -h "Cache-Control:private, max-age=0" "${publish_file_dst}" >/dev/null 2>&1 || return 1
```

Comment:
Yeah, there's another usage of Cache-Control above, and it doesn't include no-transform.  I'm not really concerned about it, since this is a text file ...